### DISCUSSION TITLE
ibax için ne yapmamız gerekiyor?

### DISCUSSION THREAD

**Discussion by literaturex** [IBAX-io/go-ibax#805] [Rewards]
ibax için ne yapmamız gerekiyor bilen birisi aydınlatırsa sevinirim.

**Comment by cryptopus58** [comment]
Tweet görevleri vs var

**Reply by literaturex** [reply]
twit görevlerini yaptım fakat puanım diğer insanlara göre az. başka da yapılacak bir şey görünmüyor
